### PR TITLE
Fix localization and button

### PR DESCRIPTION
**Pull request by Shudrum**

### CODE REVIEW

File: welcome.php [hunk 1]
```diff
@@ -52,25 +52,14 @@ public function __construct()
 
         parent::__construct();
 
-        $this->displayName = $this->l('Welcome');
-        $this->description = $this->l('Help the user to create his first product.');
+        $this->displayName = $this->getTranslator()->trans('Welcome');
+        $this->description = $this->getTranslator()->trans('Help the user to create his first product.');
```
Comment (aleeks) [inline welcome.php]: Doesn't miss domains here oO ?

File: scss/module.scss [hunk 2]
```diff
@@ -22,3 +22,44 @@
...
+      width: rem(30px);
+      height: rem(30px);
+      left: 50%;
+      top: 50%;
+      margin: {
+        top: rem(-15px);
+        left: rem(-15px);
+      }
+      display: block;
+      border: 3px solid $primary-color;
```
Comment (aleeks) [inline scss/module.scss]: rem(3px) ?

File: public/module.css [hunk 3]
```diff
@@ -295,3 +295,44 @@
...
+  .btn.with-spinner.animated:after {
+    position: absolute;
+    width: 1.875rem;
+    height: 1.875rem;
+    left: 50%;
+    top: 50%;
+    margin-top: -0.9375rem;
+    margin-left: -0.9375rem;
+    display: block;
+    border: 3px solid #25b9d7;
```
Comment (aleeks) [inline public/module.css]: rem(3px) ?

File: OnBoarding/OnBoarding.php [hunk 4]
```diff
@@ -182,14 +164,14 @@ private function loadSteps($configPath, $languageIsoCode)
      */
     private function getTextFromSettings($text)
     {
-        switch (array_keys($text)[0]) {
-            case 'loc':
-                return $this->localization[$text['loc']];
-            case 'content':
-                return $this->getTemplateContent('contents/'.$text['content']);
+        if (is_array($text)) {
+            switch ($text['type']) {
```
Comment (aleeks) [inline OnBoarding/OnBoarding.php]: We need to always have $text['type'], maybe one array_key_exist ? 
(if someday, someone change the config not formatting like you.. slash)

**Comment by AlexEven** [comment]
Hello, 

Quick info, for modules domains, we should always specify if it's admin or front content. Here it should be Modules.Welcome.Admin.
Maxime will edit this.

Thanks!

**Pull request merged**